### PR TITLE
Adding a new test case for VolumeReplication CR edit 

### PR DESCRIPTION
**Pull request by ypersky1980**
1) This PR contains a new test case for VolumeReplication CR edit 
2) In order to support creation of CR in other than openshift-storage namespace the cr_resource_not_editable function was enhanced. 
3) This PR also contain fixes of remarks by Karthick from previous pr https://github.com/red-hat-storage/ocs-ci/pull/7593

### CODE REVIEW

File: tests/e2e/system_test/test_cr_resources_validation.py [hunk 1]
```diff
@@ -105,6 +125,9 @@ def cr_resource_not_editable(
             except (
                 CommandFailed
             ):  # some properties are not editable and CommandFailed exception is thrown
+                logger.info(
+                    f"Properyy {patch} is not editable, patch command failed, continue to the next"
```
Comment (ramkiperiy) [inline tests/e2e/system_test/test_cr_resources_validation.py:129]: spell check

File: tests/e2e/system_test/test_cr_resources_validation.py [hunk 2]
```diff
@@ -154,6 +179,9 @@ def cr_resource_not_editable(
             except (
                 CommandFailed
             ):  # some properties are not editable and CommandFailed exception is thrown
+                logger.info(
+                    f"Properyy {patch} should be editable, but patch command failed, continue to the next"
```
Comment (ramkiperiy) [inline tests/e2e/system_test/test_cr_resources_validation.py:183]: spell check

File: tests/e2e/system_test/test_cr_resources_validation.py (hunk 1 above)
Comment (ypersky1980) [reply tests/e2e/system_test/test_cr_resources_validation.py]: Fixed, thank you!

**Review by ramkiperiy — DISMISSED** [review]

**Review by pintojoy — APPROVED** [review]
LGTM

File: tests/e2e/system_test/test_cr_resources_validation.py [hunk 3]
```diff
@@ -43,36 +48,49 @@ def finalizer():
...
             non_editable_patches (dict, of str: str): patches to be applied by 'oc patch' command. These patches should
                     have no effect. If such a patch is sucessfully applied, the test should fail
             editable_patches (dict, of str: str): patches to be applied by 'oc patch' command. These patches should
                     have an effect. If such a patch is not sucessfully applied, the test should fail
+            namespace (str): namespace in which cr object should be created
 
         """
-        cr_resource_yaml = os.path.join(constants.TEMPLATE_CSI_ADDONS_DIR, yaml_name)
-        res = run_oc_command(cmd=f"create -f {cr_resource_yaml}")
+        res = run_oc_command(cmd=f"create -n {namespace} -f {cr_resource_yaml}")
```
Comment (petr-balogh) [inline tests/e2e/system_test/test_cr_resources_validation.py:85]: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/performance_lib.py#L111

Why not using the namespace parameter from run_oc_command? The same in other function calls as well.

Comment (ypersky1980) [reply tests/e2e/system_test/test_cr_resources_validation.py]: You are right. Fixed it, thank you!

**Review by ramkiperiy — APPROVED** [review]

**Review by pintojoy — APPROVED** [review]
LGTM

**Pull request merged**